UIViewController-DisplayInDrawer/Classes/UIViewController+PresentDrawerController.swift

```diff
@@ -7,6 +7,14 @@ public protocol DrawerConfiguration: class {
     ///If you return nil, there will only be two positions, top and bottom
     func middlePositionY(for parentHeight: CGFloat) -> CGFloat?
     func bottomPositionY(for parentHeight: CGFloat) -> CGFloat
+    /**
+     Background color of the drawer controller. If value is nil, then light effect is applied to the background.
+     */
+    var drawerBackgroundColor: UIColor? { get }
```

Comment:
pls make this non optional and return .clear in the default implementation. Also append this comment:

Your controller _must_ have clear background color. In case you need opaque controller background color, your controller still must have clear background color, but you supply your desired opaque background color in this method. If you do not implement this method, or if you return .clear color, the default iOS-like blurred background is created for you.